src/main/kotlin/com/hostiflix/service/GithubService.kt

```diff
@@ -35,7 +35,8 @@ class GithubService (
 
             for (branch in branches) {
                 val subDomain = branch.subDomain
-                val deploymentServiceRequestDto = DeploymentServiceRequestDto(startCode, buildCode, token, gitRepo, subDomain)
+                val branchName = branch.name
```

Comment:
Can you merge this to one line? `branch.name` should be expressive enought. :smile: